payments-core/src/main/java/com/stripe/android/GooglePayJsonFactory.kt

```diff
@@ -251,17 +254,21 @@ class GooglePayJsonFactory constructor(
             .put("tokenizationSpecification", googlePayConfig.tokenizationSpecification)
     }
 
-    private fun createBaseCardPaymentMethodParams(): JSONObject {
+    private fun createBaseCardPaymentMethodParams(isReadyCall: Boolean): JSONObject {
+        val acceptedCardBrands = DEFAULT_CARD_NETWORKS
+            .plus(listOf(JCB_CARD_NETWORK).takeIf { isJcbEnabled } ?: emptyList())
+            .filter {
+                val cardBrand = networkStringToCardBrandMap[it] ?: CardBrand.Unknown
+                // Note(porter): I encountered strange behavior when using filtered card brands
```

Comment:
This seems odd, and we shouldn't need to do it. Let's get to the bottom of this.

Reply:
It's pretty odd, I spent several hours on this. If you disallow Amex on the isReady call it will error out. But none others.